source/templates.tex

```diff
@@ -5691,7 +5691,7 @@
 
 \begin{codeblocktu}{Module interface unit of \tcode{C2}}
 module;
-#include "bar.h" // imports header unit \tcode{"bar.h"}
+import "bar.h" // imports header unit \tcode{"bar.h"}
```

Comment:
This is missing the terminating semicolon. (Please also check that the comment starts in a suitable column per https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#code-blocks -- looks like we might be missing enforcement of that for `codeblocktu`.)